src/components/ha-vacuum-state.js

```diff
@@ -0,0 +1,88 @@
+import '@polymer/paper-button/paper-button.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import LocalizeMixin from '../mixins/localize-mixin.js';
+
+const STATES_INTERCEPTABLE = {
+  cleaning: {
+    action: 'Return to dock',
```

Comment:
It's unclear to me why there would be strings here? Why wouldn't these be translatable?

Reply:
You are right, I forgot to add them in, should be done now.